app/models/cloud_object_store_container/operations.rb

```diff
@@ -8,4 +8,12 @@ def delete_cloud_object_store_container
   def raw_delete
     raise NotImplementedError, _("must be implemented in subclass")
   end
+
+  def cloud_object_store_container_empty
+    raw_cloud_object_store_container_empty
+  end
+
+  def raw_cloud_object_store_container_empty
+    raise NotImplementedError, _("must be implemented in subclass")
```

Comment:
> My first impression with cloud_object_store_container_emtpy was that it was missing a question mark and it should be predicate method.

I thought the same thing when I saw `:container_empty`
I agree `clear` is more obvious especially if that is what the native method is called.